api/errors.py

```diff
@@ -0,0 +1,25 @@
+from http import HTTPStatus
+
+
+class ApiKeyError(Exception):
+    def __init__(self, message, status):
+        super().__init__()
+        self.message = message
+        self.status = status
+
+    @property
+    def json(self):
+        return {
+            'status': self.status,
+            'message': self.message,
+        }
+
+
+class AuthorizationError(ApiKeyError):
+    def __init__(self):
+        super().__init__('Invalid api key', HTTPStatus.BAD_REQUEST)
+
+
+class RequestError(ApiKeyError):
```

Comment:
Strange naming, there is nothing about ApiKey at ApiKeyError. Maybe it's better to rename ApiKeyError to just ApiError or BaseError?